Procfile

```
@@ -1,4 +1,3 @@
-release: python manage.py migrate --noinput
```

Why was this removed? Pretty useful for deploying.

---

I need to add it back.  I removed it because the release command was swallowing an error earlier in the Heroku deployment process which I've now resolved after debugging without the release process.

---

It's added back
https://github.com/SalesforceFoundation/mrbelvedereci/pull/146/commits/f26c16386357ff171fe2c0c220d03b99dbcdaccb